scripts/tutorial-patches/build.sbt.patch

```diff
@@ -21,11 +20,11 @@ index bbbb8251..b7adcb73 100644
 -  .settings(libraryDependencies ++= rocketLibDeps.value)
 -  .settings(chiselTestSettings)
 -  .settings(commonSettings)
-+//lazy val sha3 = (project in file("generators/sha3"))
-+//  .dependsOn(rocketchip, midasTargetUtils)
-+//  .settings(libraryDependencies ++= rocketLibDeps.value)
-+//  .settings(chiselTestSettings)
-+//  .settings(commonSettings)
++/* lazy val sha3 = (project in file("generators/sha3")) */
++/* .dependsOn(rocketchip, midasTargetUtils) */
++/* .settings(libraryDependencies ++= rocketLibDeps.value) */
++/* .settings(chiselTestSettings) */
++/* .settings(commonSettings) */
```

Comment:
Can you keep the same type of comments `//` instead of `/**/`? This matters for tutorial users.